vertx-config/src/main/asciidoc/index.adoc

```diff
@@ -304,7 +304,17 @@ retrieved using `config.getString(key)`.
 
 Vert.x Config can read properties file. Some properties configuration maybe is hierarchical in nature.
 This type of store allows you to convert the configuration to a json object while maintaining this hierarchy,
-in contrast to the previous method with a flat structure
+in contrast to the previous method with a flat structure.
```

Comment:
Dammit! Sorry... need to catch some sleep.

Reply:
> Can the feature be disabled? Because of backward compatibility, I believe it should be disabled by default and enabled though the config.

Good idea. Fixed